tutorials/README.md

```diff
@@ -212,54 +217,53 @@ Videos can be restricted for playback in two very distinct ways. The first is ge
 
 ##### _Tutorial Sample:_
 
-> [Restricted Video](Tutorials)
+> [Restricted Video](/tutorials)
 
 #### Deleted Videos
 
 Videos can be deleted by the content owners, for a multitude of reasons including being removed for legal or copyright violations. If you’re attempting to initialize a `Player` with content that represents deleted videos, that content is also filtered out. Only valid, playable video IDs are accepted, and have their metadata pulled into the `Player` instance. If you end up with no `Player` instance, it’s because there are no valid video IDs for it to operate on. So, you get an error to this effect. This tutorial sample shows how to detect that a video is not playable because it has been deleted.
 
 ##### _Tutorial Sample:_
 
-> [Deleted Video](Tutorials)
+> [Deleted Video](/tutorials)
```

Comment:
This is not working - the link is wrong. Please remove the `/` in all of these links. And the url probably should point to `sources` folder.

Reply:
adjust it to where you want @RomanTysiachnik